favorite-sites.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My favorite websites</title>
+	<meta charset="utf-8">
+	<meta name="Description" content=This page show my favorite websites">
+	<meta name="keywords" content="best website, best java website, , best web development website">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+	<header>
+		<h1>Nazish Ahmad</h1>
+	    <p>Hi, I'm Nazish. A computer science engineering student who love to solve problem.</p>
+	</header>
+	<main>	
+	    <h2>My Favorite Websites.</h2>	
+	    <ul>
+	    	<li>
+	    		<h3>GeeksforGeeks</h3>
+	    		<p>It is a computer science portel for geeks. GeeksforGeeks provide various courses, well written articles, interview section, quizzes etc mainly for comuter science student. Data structure is my favorite section on this website. Anyone can contribute here by sharing his article, interview experience, suggest for modification.</p>
+	    		<p> <a href="https://www.geeksforgeeks.org" target="_blank">click here</a> to reach geeksforgeeks.org</p>

```

Comment:
So this is a good time to talk about usability. 

"Click here" is never good content  to use for a button or a link -- because the user already knows to "click here". 

Instead, try a more meaningful and descriptive call to action:
```
<p> <a href="https://www.geeksforgeeks.org" target="_blank">Learn More</a> to reach geeksforgeeks.org</p> 
```

Reply:
Now I only used this;
<a href="https://www.geeksforgeeks.org" target="_blank">Learn More</a>